File: M2001/a.py
inM, inD = input().split()
inD = int(inD)

mL = {
    "JAN": 31,
    "FEB": 29,
    "MAR": 31,
    "APR": 30,
    "MAY": 31,
    "JUN": 30,
    "JUL": 31,
    "AUG": 31, 
    "SEP": 30,
    "OCT": 31,
    "NOV": 30,
    "DEC": 31
}

nM = {
    "JAN": "FEB",
    "FEB": "MAR",
    "MAR": "APR",
    "APR": "MAY",
    "MAY": "JUN",
    "JUN": "JUL",
    "JUL": "AUG",
    "AUG": "SEP", 
    "SEP": "OCT",
    "OCT": "NOV",
    "NOV": "DEC",
    "DEC": "JAN"
}

inD += 14
if (inD > mL[inM]):
    inD -= mL[inM]
    inM = nM[inM]

print(inM, inD)

File: M2001/a.cpp
#include <bits/stdc++.h>
using namespace std;

int main()
{
    int d, dm, outd;
    string m, nextm, outm;
    cin >> m >> d;
    outd = d + 14;
    
    //The Following um... lines are PURE CHEESE
    if(m=="JAN"){
        outm = "JAN";
        nextm = "FEB";
        dm = 31;
        goto brak;
    }
    if (m=="FEB"){
        outm = "FEB";
        nextm = "MAR";
        dm = 29;
        goto brak;
    }
    if (m=="MAR"){
        outm = "MAR";
        nextm = "APR";
        dm = 31;
        goto brak;
    }
    if (m=="APR"){
        outm = "APR";
        nextm = "MAY";
        dm = 30;
        goto brak;
    }
    if (m=="MAY"){
        outm = "MAY";
        nextm = "JUN";
        dm = 31;
        goto brak;
    }
    if (m=="JUN"){
        outm = "JUN";
        nextm = "JUL";
        dm = 30;
        goto brak;
    }
    if (m=="JUL"){
        outm = "JUL";
        nextm = "AUG";
        dm = 31;
        goto brak;
    }
    if (m=="AUG"){
        outm = "AUG";
        nextm = "SEP";
        dm = 31;
        goto brak;
    }
    if (m=="SEP"){
        outm = "SEP";
        nextm = "OCT";
        dm = 30;
        goto brak;
    }
    if (m=="OCT"){
        outm = "OCT";
        nextm = "NOV";
        dm = 31;
        goto brak;
    }
    if (m=="NOV"){
        outm = "NOV";
        nextm = "DEC";
        dm = 30;
        goto brak;
    }
    if (m=="DEC"){
        outm = "DEC";
        nextm = "JAN";
        dm = 31;
        goto brak;
    }
    /*
    All the goto I used are functionally 
    same as the "break"s in a switch function 
    but you can't switch string type varibles
    Q3Q
    */
    
    //Cheesing complete. now bacc to normal code
    brak:
    if(outd>dm){
        outd -= dm;
        cout << nextm << " " << outd;
    }
    else{
        cout << outm << " " << outd;
    }
}
